eng/Version.Details.xml

```diff
@@ -206,6 +206,7 @@
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21205.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>4535393d49a652c05fe7f7b2a4a752fca9ee1f45</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true"/>
```

Comment:
Are there any other dependencies that are complete that we want to add here?  xdt?  linker?

Reply:
This involved updating to a newer version and since not all dependencies are available yet, I was going to defer this until all are available and do it in one shot.